crates/ruff_python_formatter/tests/snapshots/format@statement__match.py.snap

```diff
@@ -825,6 +884,73 @@ match foo:
         **b,
     }:
         pass
+
+
+match pattern_match_class:
+    case Point2D(
+        # own line
+    ):
+        ...
+
+    case (
+        Point2D
+        # own line
+        ()
+    ):
+        ...
+
+    case Point2D(  # end of line line
+    ):
+        ...
+
+    case Point2D(
+        # end of line
```

Comment:
This is not quite right (it should be immediately after the `(`, but very hard to fix without a node for the arguments.